cookbooks/travis_worker/recipes/bluebox.rb

```diff
@@ -104,3 +104,44 @@
     notifies :run, resources(:execute => 'monit-reload')
   end
 end
+
+file '/etc/default/travis-worker-restart' do
+  content "TRAVIS_WORKER_RESTART_SLEEP=#{node['travis']['worker']['restart_sleep']}"
+  owner "root"
+  group "root"
+  mode 0644
```

Comment:
This gets rewritten every `chef-client` run, so maybe add a `not_if { File.exists?('/etc/default/travis-worker/restart') }`.